airbyte-integrations/connectors/source-jira/source_jira/manifest.yaml

```diff
@@ -0,0 +1,1335 @@
+      stop_condition: "{{ response.get('isLast') or response.get('startAt') + response.get('maxResults') >= response.get('total') }}"
+      page_size: 50
+
+  # Retriever
+
+  retriever:
+    record_selector: "#/definitions/selector"
+    requester: "#/definitions/requester"
+    paginator: "#/definitions/paginator"
+
+  retriever_no_pagination:
+    $ref: "#/definitions/retriever"
+    paginator: "#/definitions/no_pagination_paginator"
+
+  retriever_use_cache:
+    $ref: "#/definitions/retriever"
+    requester:
+      $ref: "#/definitions/retriever/requester"
+      use_cache: true
+
+  retriever_no_pagination_use_cache:
+    $ref: "#/definitions/retriever_use_cache"
+    paginator: "#/definitions/no_pagination_paginator"
+
+  retriever_v1:
+    $ref: "#/definitions/retriever"
+    requester: "#/definitions/requester_v1"
+
+  retriever_v1_use_cache:
+    $ref: "#/definitions/retriever_v1"
+    requester:
+      $ref: "#/definitions/retriever_v1/requester"
+      use_cache: true
+
+  # Service Streams
+
+  full_refresh_stream:
+    type: DeclarativeStream
+    retriever: "#/definitions/retriever"
+
+  # Full Refresh Streams
+
+  # https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-application-roles/#api-rest-api-3-applicationrole-get
+  application_roles_stream:
+    $ref: "#/definitions/full_refresh_stream"
+    name: application_roles
+    primary_key: "key"
+    retriever: "#/definitions/retriever_no_pagination"
+    $parameters:
+      path: "applicationrole"
+      extract_field: "*"
+
+  # https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-avatars/#api-rest-api-3-avatar-type-system-get
+  avatars_stream:
+    $ref: "#/definitions/full_refresh_stream"
+    name: avatars
+    primary_key: "id"
+    retriever:
+      $ref: "#/definitions/retriever_no_pagination"
+      partition_router:
+        type: ListPartitionRouter
+        cursor_field: "slice"
+        values:
+          - issuetype
+          - project
+          - user
+    $parameters:
+      path: "avatar/{{ stream_slice.slice }}/system"
+      extract_field: "system"
+
+  # https://developer.atlassian.com/cloud/jira/software/rest/api-group-other-operations/#api-agile-1-0-board-get
+  boards_stream:
+    $ref: "#/definitions/full_refresh_stream"
+    name: boards
+    primary_key: "id"
+    retriever:
+      $ref: "#/definitions/retriever_v1_use_cache"
+      record_selector:
+        $ref: "#/definitions/selector"
+        record_filter:
+          condition: "{{ not config.get('projects') or record.get('location', {}).get('projectKey') in config['projects'] }}"
+    transformations:
+      - type: AddFields
+        fields:
+          - path: ["projectId"]
+            value_type: string
+            value: "{{ record.get('location', {}).get('projectId', '') }}"
+          - path: ["projectKey"]
+            value_type: string
+            value: "{{ record.get('location', {}).get('projectKey', '') }}"
+    $parameters:
```

Comment:
Agree, but to avoid blocking the PR I created an issue: https://github.com/airbytehq/airbyte-internal-issues/issues/7604. I will do refatctoring and remove $parameters from the manifest later in current or next sprint